web/src/features/prompts/server/actions/getPromptsMeta.ts

```diff
@@ -0,0 +1,89 @@
+import { type GetPromptsMetaType } from "@/src/features/prompts/server/utils/validation";
+import { Prisma, prisma } from "@langfuse/shared/src/db";
+
+export type GetPromptsMetaParams = GetPromptsMetaType & { projectId: string };
+
+export const getPromptsMeta = async (
+  params: GetPromptsMetaParams,
+): Promise<PromptsMetaResponse> => {
+  const { projectId, page, limit } = params;
+
+  const promptsMeta = (await prisma.$queryRaw`
+    SELECT
+        p.name AS "promptName",
+        p.tags AS tags,
+        array_agg(DISTINCT p.version) AS versions,
+        array_agg(DISTINCT label) AS labels
+    FROM
+        prompts p
+    LEFT JOIN LATERAL unnest(p.labels) AS label ON true
```

Comment:
do we have to do this with a lateral join? Cant we unnest somehow in the select?